app/assets/javascripts/preview.js

```diff
@@ -9,22 +9,30 @@ $(function() {
                 '</tr>');
         }
     });
+    var generating_preview_version = false;
```

Comment:
We should use `let` instead of `var`.
```suggestion
    let generating_preview_version = false;
```

Reply:
changed